README.md

```diff
@@ -1,30 +1,64 @@
+        'HOST': '127.0.0.1', # The IP address of the database server. If it is the same server, this can be left as is
+        'PORT': '5432', # The port the database server is running on. It is unlikely you will need to change this
+    }
+}
+```
 
 # Installation - without Docker
+#### Not officially supported. It is possible, but we aren't able to assist with the installation if you choose to do so.
+
 1. Clone the repository! You need the code, it won't just magically work without it.
-2. Install requirements - ensure you already have python3 installed on your machine. We recommend the latest python3.7 build
-```pip install -r requirements.txt``` (run that command, assuming you are in the main directory
-3. Get a secret key! By default there is no secret key, so be sure to put a secret key in settings.py -
-the following lines will have django create a new key for you
-`from django.core.management.utils import get_random_secret_key`
-`get_random_secret_key()`
-4. Apply database migrations by running ```python3 manage.py migrate``` (assuming your database options are set as you'd like in settings.py)
-5. If you are utilizing AWS for storage, make sure you obtain the required keys and put them in settings.py
-6. If you are looking to enable CAPTCHA for logins, make sure you obtain the required keys and put them in settings.py 
-as well as profiles/templates/captcha.html - and uncomment the lines in profiles.views that pertain to form verification of the captcha success
-7. Be sure to set SMTP email settings in settings.py so account activation emails can be sent out to new users
-8. Create a django admin/super user account - ```python manage.py createsuperuser```
-9. Give your new superuser account staff panel access by modifying the database row in the `profiles_userprofile` table - set
-`user_verified` to `1` and set `user_type` to `superadmin` or `admin`
-9. Create a StaticInfo object, if you don't every page will throw an error telling you that it can't access the StaticInfo - easiest
+2. Install requirements - `pip install -r requirements.txt`
+3. Configure settings.py
+4. Apply database migrations by running `python3 manage.py migrate` (assuming your database options are set as you'd like in settings.py)
+5. Create a django admin/super user account - `python manage.py createsuperuser` and follow the prompts
+6. Give your new superuser account staff panel access by modifying the database row in the `profiles_userprofile` table - set
+`user_verified` to `1` and set `user_type` to `superadmin` or `admin` (hint: you can use django admin at /admin)
+7. Create a StaticInfo object, if you don't every page will throw an error telling you that it can't access the StaticInfo - easiest
 way to make a staticinfo object is through the django admin at /admin
+8. Setup a web server and reverse proxy using industry standard best practices (hint: gunicorn and nginx work well)
 
-# Docker Instalation
```

Comment:
nice job here buddy

Reply:
Fixed in 98ce8e53ad699b313fc7b3b65f76d18790d83b4c